alerts/apps/notifications/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+class Task(models.Model):
+
+    title = models.CharField('Заголовок задачи', max_length = 100)
```

Comment:
обычно (согласно какому-то пепу) именованные аргументы при передаче пишутся в формате `arg_name=<value>`, то есть `=` без окружающих пробелов. Поэтому правильнее:
```suggestion
    title = models.CharField('Заголовок задачи', max_length=100)
```

В остальных случаях ниже тоже лучше переделать, я не буду там комменты оставлять

Reply:
переделал